index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <script defer src="./scripts.js"></script>
+
+    <title>Avengers Conference</title>
+</head>
+
+<body>
+    <header class="mobile-header">
```

Comment:
- [ ] The Navbar should be divided into a top dark bar and a bottom white bar so your design is identical to the CC design
  - [ ] Please consider implementing a top short bar section with social media links, logout, and language links aligned to the right
  - [ ] Good work implementing the taller bottom bar with a logo and a list of page links but you can consider decreasing the padding and the logo size so your design is similar to the given one
#### Please implement the changes in the ```About page ``` also